### DISCUSSION TITLE
Multitenancy with Linkerd

### DISCUSSION THREAD

**Discussion by jnsrikanth** [linkerd/linkerd2#4935] [General]
How do I achieve Software Namespace based Multi-Tenancy using Linkerd?

**Comment by ihcsim** [comment]
Linkerd doesn't support namespace multi-tenancy. What problem are you trying to solve?

**Comment by jnsrikanth** [comment]
Hi
I am trying to create a multi-tenancy architecture for serving two clients of mine who need to be served with different versions and a different configuration/version of my application as well as different database schema but from the same cluster, I created using Kubernetes. 

However, I am quite impressed with the many out of box features that linkerd provides from the control plane and I would like to make use of such features in a multi-tenant environment.
Please let me know. Thanks.

-Srikanth

    On Tuesday, September 1, 2020, 10:00:41 PM GMT+5:30, Ivan Sim <notifications@github.com> wrote:  
 
 


Linkerd doesn't support namespace multi-tenancy. What problem are you trying to solve?

**Reply by ihcsim** [reply]
You won't be able to install multiple instances of Linkerd on the same K8s cluster, due to cluster-scoped resources like admission webhooks and api extensions, which don't play well with multiple Linkerd control planes.
